packages/react-core/src/components/Progress/Progress.tsx

```diff
@@ -81,24 +84,33 @@ export class Progress extends React.Component<ProgressProps> {
       isTitleTruncated,
       tooltipPosition,
       'aria-label': ariaLabel,
+      'aria-labelledby': ariaLabelledBy,
       ...props
     } = this.props;
 
     const progressBarAriaProps: AriaProps = {
-      'aria-labelledby': title ? `${this.id}-description` : null,
-      'aria-label': ariaLabel ? ariaLabel : null,
       'aria-valuemin': min,
       'aria-valuenow': value,
       'aria-valuemax': max
     };
 
+    if (title || ariaLabelledBy) {
+      progressBarAriaProps['aria-labelledby'] = title ? `${this.id}-description` : ariaLabelledBy;
+    }
+
+    if (ariaLabel) {
+      progressBarAriaProps['aria-label'] = ariaLabel;
+    }
+
     if (valueText) {
       progressBarAriaProps['aria-valuetext'] = valueText;
     }
 
-    if (!title && !this.props['aria-label']) {
+    if (!title && !ariaLabelledBy && !ariaLabel) {
       /* eslint-disable no-console */
-      console.warn('An accessible aria-label is required when using the progress component without a title.');
```

Comment:
can you add a jest test to verify console warning?

Reply:
I think there is already one, see: 'progress component generates console warning when no accessible name is provided'